src/test/java/org/broadinstitute/hellbender/tools/exome/PlotACNVResultsIntegrationTest.java

```diff
@@ -20,7 +20,7 @@
     public void testACNVPlotting() {
         File tDir = IOUtils.tempDir("Test", "Plotting");
         final String[] arguments = {
-                "-" + ExomeStandardArgumentDefinitions.SNP_FILE_SHORT_NAME, SNP_COUNTS_FILE.getAbsolutePath(),
+                "-" + ExomeStandardArgumentDefinitions.ALLELIC_COUNTS_FILE_SHORT_NAME, SNP_COUNTS_FILE.getAbsolutePath(),
```

Comment:
it's a simulated tumor pulldown.  i went ahead and renamed it to TUMOR_ALLELIC_COUNTS_FILE throughout.